src/library-authoring/data/apiHook.ts

```diff
@@ -15,6 +15,40 @@ export const useContentLibrary = (libraryId?: string) => (
   })
 );
 
+/**
+ *  Hook to fetch block types of a library.
+ */
+export const useLibraryBlockTypes = (libraryId) => (
+  useQuery({
+    queryKey: ['contentLibrary', 'libraryBlockTypes', libraryId],
```

Comment:
This should be using a [query key factory](https://github.com/openedx/frontend-app-course-authoring/blob/cc41a2fda1206e36fb5a5fd92d9242b3a4b1c5ee/src/taxonomy/data/apiHooks.js#L20-L66). But note I told @rpenido the same thing in a recent PR of his.

In addition, this key is in the wrong order. It should be `['contentLibrary', libraryId, 'content', 'libraryBlockTypes'],` because when we add/remove a block from the library, we want to be able to invalidate all queries for the content in the library, e.g. `['contentLibrary', libraryId, 'content', *]` and it's nice to do that with a single invalidation.